modules/conflict_resolver/php/conflict_resolver.class.inc

```diff
@@ -236,7 +236,7 @@ class Conflict_Resolver extends \NDB_Menu_Filter_Form
             LEFT JOIN session ON (flag.SessionID=session.ID)
             LEFT JOIN candidate ON (candidate.CandID=session.CandID)
             LEFT JOIN Project ON (candidate.ProjectID=Project.ProjectID )
-            WHERE 1=1";
+            WHERE 1=1 AND candidate.Active ='Y'";
```

Comment:
You can take out the 1=1. That's just there to ensure that if filters add "AND" clauses it doesn't become an SQL syntax error but is unnecessary if there's already a WHERE clause in the query.